include/wlr/types/wlr_input_device.h

```diff
@@ -9,6 +9,8 @@
 #ifndef WLR_TYPES_WLR_INPUT_DEVICE_H
 #define WLR_TYPES_WLR_INPUT_DEVICE_H
 
+#include "wayland-server-core.h"
```

Comment:
This should use angle brackets since it's an external header.

Reply:
Thanks for the catch